server/src/models/student.ts

```diff
@@ -133,4 +137,12 @@ export class Student {
 
   @Column({ nullable: true, default: true })
   mentoring: boolean;
+
+  @ManyToMany(_ => TeamDistribution, teamDistribution => teamDistribution.students)
```

Comment:
Argument for first callback here is not needed, I would prefer to make it a bit more clear
```suggestion
  @ManyToMany(() => TeamDistribution, teamDistribution => teamDistribution.students)
```
Same for other similar stuff

Reply:
Thank you, fixed